### PR TITLE
make_dataset (tokenizing pending)

### PR DESCRIPTION
**Pull request by LauraPaz**

### CODE REVIEW

File: src/data/make_dataset.py [hunk 1]
```diff
@@ -1,3 +1,79 @@
+import os
+import zipfile
+from typing import Union
+import pandas as pd
+from kaggle.api.kaggle_api_extended import KaggleApi
+from sklearn.model_selection import train_test_split
+
+def download_dataset(dataset_name: str, destination_path: str) -> None:
```
Comment (truszkowski-tomasz) [inline src/data/make_dataset.py]: If we want to store data in google drive maybe it is not necessary to download it through kaggle? Once we all have access to google drive, each of us can easily download the data from there.

So my suggestion would be to download the raw data, add it to the folder, configure dvc, push changes, send link to others. Does it make sense? What do you think?

**Review by truszkowski-tomasz — APPROVED** [review]

File: src/data/make_dataset.py (hunk 1 above)
Comment (LauraPaz) [reply src/data/make_dataset.py]: Makes sense, I'll configure dvc and remove the kaggle code.

File: .dvc/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/config.local
```
Comment (truszkowski-tomasz) [inline .dvc/.gitignore:1]: Can we add them to global '.gitignore', add only 1 file?

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,5 @@
-torch >= 2.1.0
-transformers >= 4.36.0
\ No newline at end of file
+kaggle==1.6.0
```
Comment (truszkowski-tomasz) [inline requirements.txt:1]: No more need for that, right?

**Review by truszkowski-tomasz — APPROVED** [review]
Looks good to me:)

**Pull request closed without merging**